### PR TITLE
Forth input for sleeper

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
New input added: Distance to bed (meters). Before starting to sleep, it will walk the given distance

## Checklist

- [ ] I think the code is well written
- [ ] I updated the table of contents (make toc)
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/sleeper/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 
+## [2.1.1] - 2020-02-24
+### Added
+- Forth input added, "Distance to bed" (meters). Before sleeping, it will walk this distance
```
Comment (KZzizzle) [inline services/sleeper/CHANGELOG.md:12]: Fourth

File: services/sleeper/src/sleeper/main.py [hunk 2]
```diff
@@ -50,6 +50,16 @@ def test_gpu_cuda_code() -> None:
     # search the history for the CUDA implementation
 
 
+def walk_to_bed(
+    amount_to_walk: int = 0
+) -> None:
+    if amount_to_walk > 0:
+        print(f"So tired, I first need to walk {amount_to_walk} meters to bed")
```
Comment (KZzizzle) [inline services/sleeper/src/sleeper/main.py:57]: amazing

**Review by KZzizzle — APPROVED** [review]
so sleepy

**Review by mguidon — APPROVED** [review]
Your sleeper gets quite opinionated. Why would you want to sleep in a bed? Here, beyond the wall, we sometimes sleep naked outside under the moon.
Furthermore, what does it even mean "It will first walk the distance to bed". In order to know how long that takes you have to provide additional inputs, the distance, and, obviously the average walking speed. This might actually be a good exercise in units conversion. If the sleeper is us-american, he might not even know what a meter is.
But other than that, I approve.

**Comment by odeimaiz** [comment]
> Your sleeper gets quite opinionated. Why would you want to sleep in a bed? Here, beyond the wall, we sometimes sleep naked outside under the moon.
> Furthermore, what does it even mean "It will first walk the distance to bed". In order to know how long that takes you have to provide additional inputs, the distance, and, obviously the average walking speed. This might actually be a good exercise in units conversion. If the sleeper is us-american, he might not even know what a meter is.
> But other than that, I approve.

I took the average tired-but-really-want-to-go-to-bed walking pace: 1m/s, 2 steps per second.

File: services/sleeper/src/sleeper/main.py [hunk 3]
```diff
@@ -78,11 +88,14 @@ def main() -> None:
     1 and 9 will be used.
 
     INPUT_3 will cause this script to fail after sleeping.
+
+    Before sleeping, it will walk first the distance given in INPUT_4.
```
Comment (GitHK) [inline services/sleeper/src/sleeper/main.py:92]: 😆

**Review by GitHK — APPROVED** [review]

**Pull request merged**